joltify/assetlist.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "joltify",
+  "assets": [
+    {
+      "description": "The native staking and governance token of Joltify",
+      "extended_description": "Joltify is the First EVM compatible game-changing L1 Public Chain built on the Cosmos SDK for Real-World Assets (RWA) and Beyond.",
+      "denom_units": [
+        {
+          "denom": "ujolt",
+          "exponent": 0
+        },
+        {
+          "denom": "jolt",
+          "exponent": 6
+        }
+      ],
+      "base": "ujolt",
+      "name": "Joltify",
+      "display": "jolt",
+      "symbol": "JOLT",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.svg"
+        }
+      ],
+      "coingecko_id": "joltify",
+      "keywords": [
+        "staking"
+      ],
+      "socials": {
+        "website": "https://www.joltify.io/",
+        "twitter": "https://twitter.com/joltify_finance"
+      }
+    },
+    {
+      "description": "The native staking and governance token of the Cosmos Hub.",
+      "denom_units": [
+        {
+          "denom": "ibc/9117A26BA81E29FA4F78F57DC2BD90CD3D26848101BA880445F119B22A1E254E",
+          "exponent": 0,
+          "aliases": [
+            "uatom"
+          ]
+        },
+        {
+          "denom": "atom",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/9117A26BA81E29FA4F78F57DC2BD90CD3D26848101BA880445F119B22A1E254E",
+      "name": "Cosmos Hub",
+      "display": "atom",
+      "symbol": "ATOM",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "cosmoshub",
+            "base_denom": "uatom",
+            "channel_id": "channel-864"
```

Comment:
Please register this IBC connection

Reply:
and the IBC connection for each registered asset (so, including joltify<>noble because of USDC)

Reply:
Hey Jeremy, could I ask where should I go to register the opened IBC connections? Any link to the corresponding repos?

Reply:
is the following repo correct?
https://github.com/cosmos/ibc

Reply:
No, it would be here: https://github.com/cosmos/chain-registry/tree/master/_IBC